main.js

```diff
@@ -153,8 +153,11 @@ module.exports = function (argumentOptions) {
         connectionOptions.routes = options.routes;
     }
 
-    if (options.state) {
-        connectionOptions.state = options.state;
+    if (_.get(options, 'connections.routes.files', false)) {
+        server.settings.connections.routes.files = options.connections.routes.files;
+    }
+    if (_.get(options, 'connections.routes.state', false)) {
+        server.settings.connections.routes.state = options.connections.routes.state;
```

Comment:
Yes I agree.  I'll update to that.  I don't think it will cause any conflicts from what I saw.  Was being cautious (specific) in case there was something I wasn't aware of yet.